src/libraries/apicompat/ApiCompat.proj

```diff
@@ -7,6 +7,8 @@
     <!-- Target reference assemblies instead of implementation assemblies. -->
     <CompileUsingReferenceAssemblies>true</CompileUsingReferenceAssemblies>
     <TrimOutPrivateAssembliesFromReferencePath>true</TrimOutPrivateAssembliesFromReferencePath>
+    <!-- Temporarily disable baseline validation so that we can add baselines for the upcoming API compat fix https://github.com/dotnet/arcade/pull/9168 -->
```

Comment:
This flag makes sure that our baseline files are *minimal*.  That fails right now because the current APICompat doesn't generate any of these added errors.  I'll disable that checking until we get the API compat update.

Reply:
Can you please add a tracking issue for that? In the past we often forgot to turn such knobs on again.